src/main/java/org/apache/commons/compress/archivers/tar/TarConstants.java

```diff
@@ -347,6 +361,13 @@
      */
     String MAGIC_XSTAR = "tar\0";
 
+    /**
+     * Offset inside the header for the xtar multivolume data
+     *
```

Comment:
Is there an RFC or resource for which we can provide a URL for these offsets?

Reply:
I derived it from [star's source code](https://sourceforge.net/projects/s-tar/files/star-1.6.tar.bz2). Specifically, `star.h`, line 400, which you can also view [here](https://fossies.org/linux/misc/star-1.6.tar.bz2/star-1.6/star/star.h).

I also added the struct and the offsets at the javadoc for `TarArchiveEntry`. Would you like me to find a way to incorporate a link or pointer to the source code?

Reply:
If you've already documented well on one place you can just use "@see" to that place.

Reply:
I see there's no support for multivolume TAR archives yet. 

I decided to only add the `@see` bits in the offset constants I added and to add a note on the multivolume check for xstar for clarity. The whole section was taken from star anyway, so I trust it is correct. See 07d34a2.

Regardless, the `M` link flag is a GNU/xstar-specific extension that's only used in multivolume archives, which compress currently doesn't support, so that code is there more for documentation/future-proofing than anything else.